mix.exs

```diff
@@ -46,4 +46,4 @@ defmodule Nerves.Firmware.HTTP.Mixfile do
         { :exjsx, "~> 3.2.0" },
         { :ex_doc, "~> 0.11", only: :dev} ]
     end
-end
\ No newline at end of file
+end
```

Comment:
My text editor automatically adds a new line at the end of the file. Not sure I can easily get around it.